auto-ipsec/racoon/README.md

```diff
@@ -30,7 +30,7 @@ In addition to basic keylime setup, the following configuration options must be
 
 ### keylime.conf on each agent
 
-to support automatic revocation, the revocation notifier must be enabled and reachable by agents.  Set the IP/port to the verifier that hosts the revocation notifier:
+To support automatic revocation, the revocation notifier must be enabled and reachable by agents.  Set the IP/port to the verifier that hosts the revocation notifier:
```

Comment:
This was changed to "... reachable by machines running the Keylime agent" in libreswan